<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filterwidget</class>
 <widget class="QWidget" name="Filterwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <widget class="QTextEdit" name="TextEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>571</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TipsLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>76</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Tips: One for each line</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>581</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RBG1">
    <item>
     <widget class="QRadioButton" name="PathRB">
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="NameRB">
      <property name="text">
       <string>Filename</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="TypeRB">
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="TimeRB">
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>371</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RBG2">
    <item>
     <widget class="QRadioButton" name="BlackRB">
      <property name="text">
       <string>Blacklist</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="WhiteRB">
      <property name="text">
       <string>Whitelist</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ConfirmButton">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>295</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="TimeLO">
    <item row="0" column="0">
     <widget class="QLabel" name="FromLabel">
      <property name="text">
       <string>from</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="FromTime">
      <property name="currentSection">
       <enum>QDateTimeEdit::YearSection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="currentSectionIndex">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Tolabel">
      <property name="text">
       <string>to</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="ToTime">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
